projects/commudle-admin/src/app/components/home/home.component.html

```diff
@@ -12,7 +12,9 @@
         <app-home-external-feed-links></app-home-external-feed-links>
       </div>
       <div class="right-column">
-        <app-home-promotions></app-home-promotions>
+        <div class="wrapper" >
+          <app-home-promotions class="wrapper__img" ></app-home-promotions>
+        </div>
```

Comment:
rename class to: promotion or promotion_container

remove class from app-home-promotions

Reply:
Done!